adminsortable/admin.py

```diff
@@ -297,8 +297,11 @@ def do_sorting_view(self, request, model_type_id=None):
 
                 for index in indexes:
                     obj = objects_dict.get(index)
-                    setattr(obj, order_field_name, start_index)
-                    obj.save()
+                    # perform the update iff the order field has changed

```

Comment:
I clarified the comment.  No problem, happy to contribute.  Thanks for this project!